src/vs/workbench/contrib/comments/browser/commentsView.ts

```diff
@@ -391,8 +400,16 @@ export class CommentsPanel extends FilterViewPane implements ICommentsView {
 			overrideStyles: this.getLocationBasedColors().listOverrideStyles,
 			selectionNavigation: true,
 			filter: this.filter,
+			sorter: {
+				compare: (a: CommentsTreeNode, b: CommentsTreeNode) => {
+					if (this.filters.sortBy === CommentsSortOrder.UpdatedAtDescending && 'lastUpdatedAt' in a && 'lastUpdatedAt' in b) {
+						return a.lastUpdatedAt > b.lastUpdatedAt ? -1 : 1;
```

Comment:
Forgot to mention this one in my review: I don't think the `in` check would survive a refactoring if `lastUpdatedAt` gets renamed. I'll change it to an `instanceof` check.